builder.config.json

```diff
@@ -199,7 +206,23 @@
     "isEditableLocked": 0,
     "orgTimezone": "America/Toronto",
     "subscriberCount": 0,
-    "subscriberPropertyFields": [],
+    "subscriberPropertyFields": [
```

Comment:
@bzabos , I have the same question here like on https://github.com/FLOIP/flow-builder/pull/43/files#diff-7408c303e8b65d099dc2c7e2379e01d3cca5356afc39646a8e1d6099dde46ecbR253

Reply:
@safydy and I discussed that data livinng in our default `builder.config.json` should be data that will be helpful to _most_ developers consuming+testing this repository. If we'd like to provide examples of how components will look with different types of data, those can live inside storybook stories. 

@jcbashdown I love what you're suggesting there— We can have a developer experience that loads in some sample flows 👌

Reply:
I removed sample data from here, and created a new story